app-connector-client/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node server/server",
     "start:dev": "export DEBUG=true && node server/server test/varkes_config.json",
-    "test": "mocha"
+    "test": "nyc mocha --exit"
```

Comment:
Why is it needed to hard exit the test, it seems that the test execution is not ending smooth